### PR TITLE
Add support for years

### LINKED ISSUE #16: Add support for years
```js
var prettyMs = require('pretty-ms');
prettyMs(133700000000);

//"1547d 10h 53m 20s"
```

Days should be in year at least

### PR DESCRIPTION
**Pull request by icyflame**
Fixes #16 

@sindresorhus I hope you were talking about something like this in your last comment where you consented to including years in the output.

### CODE REVIEW

**Comment by icyflame** [comment]
@sindresorhus A lot has changed in the JS world, in the last 2 years, it seems like. 

If you want to make this module completely ES6, then we should convert all `var` to `let` and `const`. :slightly_smiling_face:

**Comment by sindresorhus** [comment]
> If you want to make this module completely ES6, then we should convert all `var` to `let` and `const`. 🙂

Yes, I will do that, but that's outside the scope of this PR.

File: index.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ module.exports = function (ms, opts) {
 
 	var parsed = parseMs(ms);
 
-	add(parsed.days, 'day', 'd');
+	add(parseInt(parsed.days / 365, 10), 'year', 'y');
```
Comment (sindresorhus) [inline index.js]: `Math.trunc` would be better here.

File: test.js [hunk 2]
```diff
@@ -16,11 +16,15 @@ it('should prettify milliseconds', function () {
 	assert.strictEqual(prettyMs(1000 * 60 * 60 * 12), '12h');
 	assert.strictEqual(prettyMs(1000 * 60 * 60 * 40), '1d 16h');
 	assert.strictEqual(prettyMs(1000 * 60 * 60 * 999), '41d 15h');
+	assert.strictEqual(fn(1000 * 60 * 60 * 24 * 465), '1y 100d');
+	assert.strictEqual(fn(1000 * 60 * 67 * 24 * 465), '1y 154d 6h');
```
Comment (sindresorhus) [inline test.js]: Should be named `prettyMs`.

File: test.js [hunk 3]
```diff
@@ -16,11 +16,15 @@ it('should prettify milliseconds', function () {
...
 });
 
 it('should have a compact option', function () {
 	assert.strictEqual(prettyMs(1000 + 4, {compact: true}), '~1s');
 	assert.strictEqual(prettyMs(1000 * 60 * 60 * 999, {compact: true}), '~41d');
+	assert.strictEqual(fn(1000 * 60 * 60 * 24 * 465, {compact: true}), '~1y');
+	assert.strictEqual(fn(1000 * 60 * 67 * 24 * 465, {compact: true}), '~1y');
```
Comment (sindresorhus) [inline test.js]: Here too.

**Comment by icyflame** [comment]
All changes done.

> Yes, I will do that, but that's outside the scope of this PR.

@sindresorhus Thanks a lot, I was worried about the failing test cases. :slightly_smiling_face:

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**